FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/AzureStorageJobFactory.cs

```diff
@@ -124,12 +128,14 @@ private IJob CreateOrchestratorTask(JobInfo jobInfo)
                         _metadataStore,
                         _maxJobCountInRunningPool,
                         _metricsLogger,
+                        _jobExecutionErrorProcessor,
                         _diagnosticLogger,
                         _loggerFactory.CreateLogger<FhirToDataLakeOrchestratorJob>());
                 }
             }
             catch (Exception e)
             {
+                _metricsLogger.LogTotalErrorsMetrics(ErrorType.CreateJobError, $"Failed to create orchestrator job. Reason: {e.Message}", Operations.CreateJob);
```

Comment:
Is it possible to use `_jobExecutionErrorProcessor` here since seems it already contains the metrics reporting mechanism.

Reply:
Added a new processor for job factory.
_jobExecutionErrorProcessor is only used for job running stage's exceptions.